lib/evmone/instructions_traits.hpp

```diff
@@ -183,6 +183,12 @@ struct Traits
     /// The stack height change caused by the instruction execution. Can be negative.
     int8_t stack_height_change = 0;
 
+    /// Size of the immediate argument in bytes.
+    uint8_t immediate_size = 0;
```

Comment:
Probably better to place it as the second field. The whole traits change should be submitted separately.

Reply:
Why th second? What about `is_terminating`?

Do you want a separate PR? (these fields are used only in this PR) I put it into a separate commit so far.